src/functions/durableEntity.ts

```diff
@@ -0,0 +1,47 @@
+import { app, HttpHandler, HttpRequest, HttpResponseBody, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { EntityHandler } from 'durable-functions';
+
+// Replace with your own Durable entity name
+const entityName = 'Counter';
+
+const clientInput = df.input.client();
+
+const clientHandler: HttpHandler = async (context: InvocationContext, req: HttpRequest) => {
+    const id: string = req.params.id;
+    const entityId = new df.EntityId(entityName, id);
+    const client = df.getClient(context, clientInput);
+
```

Comment:
I just realized: there's only 1 type of DF input bindings: the `durableClient`. 

With that in mind, do we really need this second parameter? Can we not just call `df.getClient(context)` and have the method always retrieve the `durableClient` within?

Reply:
> With that in mind, do we really need this second parameter?

Sadly, yes. Under the hood, `df.getClient()` calls `context.getInput()`, which expects an input binding as an argument (`clientInput` in this case), so the functions needs to also pass that argument to the SDK. In the old model, this wasn't as necessary because all the bindings would live inside the `context`, so the SDK could just loop over all of them until it finfs the client input binding, but that is no longer the case in the new model

Reply:
I see. At this time, this is my biggest pain point with the new programming model. Moving forward, I would like for us to prioritize removing this boilerplate, as it is rather clunky and DF apps always need a client.

Reply:
I agree with this 100%. But I think we agreed that that shouldn't be in this first iteration. Let me create an issue for this on the Durable SDK to track this so we don't block this PR.

Reply:
> In the old model, this wasn't as necessary because all the bindings would live inside the context, so the SDK could just loop over all of them until it finds the client input binding, but that is no longer the case in the new model

Ftr, the new model _could_ do the same thing where it loops over the context to find the durable client. The library package might not allow you to loop over bindings right now, but we can always change it. That may be helpful outside of durable as well, which you should always keep an eye out for.

If we can come up with a fancy new design to _improve_ on the client experience, that's great - but in the interest of time we should keep the "match the old model" option on the table.